_pages/sn95.yaml

```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: sn95
+pageUrl: sn95.github.io
+timestamp: 2017-12-13
+---
```

Comment:
@sn95 you just need a line break after line 5!

Reply:
Thank you!